modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -254,6 +273,60 @@ PortletConfigurationPermissionPropagation portletConfigurationPermissionPropagat
 		'<portlet:namespace />saveButton'
 	);
 
+	var <portlet:namespace />permissionPropagationEnabledCheckbox = document.getElementById(
+		'<portlet:namespace />permissionPropagationEnabled'
+	);
+
+	var <portlet:namespace />undoButton = document.getElementById(
+		'<portlet:namespace />undoButton'
+	);
+
+	var alertMessage = document.getElementById('<portlet:namespace />alertMessage');
+
+	if (<portlet:namespace />permissionPropagationEnabledCheckbox) {
+		var initialPermissionPropagationEnabled = Boolean(
+			<%= portletConfigurationPermissionPropagation.getPermissionPropagation(renderRequest) %>
+		);
```

Comment:
Could it be like that?
`var initialPermissionPropagationEnabled = <portlet:namespace />permissionPropagationEnabledCheckbox.checked;`